pkg/api/unversioned/generated.proto

```diff
@@ -232,6 +232,24 @@ message ListMeta {
   optional string resourceVersion = 2;
 }
 
+// MicroTime is version of Time with microsecond level precision.
+// 
+// +protobuf.options.marshal=false
+// +protobuf.as=Timestamp
+// +protobuf.options.(gogoproto.goproto_stringer)=false
+message MicroTime {
```

Comment:
Isn't this ~= https://github.com/google/protobuf/blob/master/src/google/protobuf/timestamp.proto 

Just wondering why we're duping.